Sources/SwiftProcessing/Core/Sketch.swift

```diff
@@ -263,20 +263,53 @@ import GameplayKit
     
     public var key: Character = "\0"
     
+    private var _internalKey: Any? = nil
     @available(iOS 13.4, *)
-    lazy var internalKey: UIKey = UIKey()
+    fileprivate var internalKey: UIKey {
+        get {
+            if _internalKey == nil {
+                _internalKey = UIKey()
+            }
+            return _internalKey as! UIKey
+        }
+        set {
+            _internalKey = newValue
+        }
+    }
     
     /// The boolean system variable keyPressed is true if any key is pressed and false if no keys are pressed. Note that there is a similarly named function called keyPressed(). See its reference page for more information.
     
+    private var _keyPressed: Any? = nil
     @available(iOS 13.4, *)
-    public lazy var keyPressed = false
+    public var keyPressed: Bool {
+        get {
+            if _keyPressed == nil {
+                _keyPressed = false
+            }
+            return _keyPressed as! Bool
+        }
+        set {
+            _keyPressed = newValue
+        }
+    }
     
     /// The variable keyCode is used to detect special keys such as the UP, DOWN, LEFT, RIGHT arrow keys and ALT, CONTROL, SHIFT.
     /// When checking for these keys, it can be useful to first check if the key is coded. This is done with the conditional if (key == CODED), as shown in the example above.
     /// The keys included in the ASCII specification (BACKSPACE, TAB, ENTER, RETURN, ESC, and DELETE) do not require checking to see if the key is coded; for those keys, you should simply use the key variable directly (and not keyCode). If you're making cross-platform projects, note that the ENTER key is commonly used on PCs and Unix, while the RETURN key is used on Macs. Make sure your program will work on all platforms by checking for both ENTER and RETURN.
     
+    private var _keyCode: Any? = nil
     @available(iOS 13.4, *)
-    public lazy var keyCode = KeyCode.none
+    public var keyCode: KeyCode {
+        get {
+            if _keyCode == nil {
+                _keyCode = KeyCode.none
+            }
+            return _keyCode as! KeyCode
+        }
+        set {
+            _keyCode = newValue
```

Comment:
I wonder if we're losing type checking here: a crash won't happen on the `set`, it'll happen on the `get` during the forced type casting?

Reply:
I might be overthinking it actually. Since `keyCode` is set to be `KeyCode` I assume there would be a compile time error if we tried setting it to something that wasn't `KeyCode` before getting here...?